test/integration/dialects/mysql/errors.test.js

```diff
@@ -0,0 +1,81 @@
+'use strict';
+
+const chai = require('chai'),
+  expect = chai.expect,
+  Support = require(__dirname + '/../../support'),
+  dialect = Support.getTestDialect(),
+  DataTypes = require(__dirname + '/../../../../lib/data-types')
+  ;
+
+if (dialect === 'mysql') {
+  describe.only('[MYSQL Specific] Errors', () => {
```

Comment:
no `only`, I have add a rule in eslint config to prevent this in future https://github.com/sequelize/sequelize/commit/3e0c9b1acfd2538d7a155e6c3b3783938121bcd6

Reply:
oops. sorry - my bad. I've removed it.